### ISSUE TITLE
Enable fast acessibility by default, rest behind persistent option

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/joist#58]
It should be possible to have the "fast" parts of accessibility enabled by default, but have the "slow" things (like moving the peer divs to match positions so zooming works well) behind an option that defaults to off, but is persistent (saved across browsing sessions).

We should test to determine what is 'fast' and what is 'slow'. @samreid suggested the possibility of only enabling peers on the menu -> options -> accessibility so that we don't create all peers by default (may be best for performance). Will need to test FPS with and without certain types of accessibility.

**Comment by jessegreenberg** [comment]
https://github.com/phetsims/scenery/issues/41 will neeed to be completed before we handle this issue.

**Comment by ariel-phet** [comment]
@jessegreenberg what is the status of this issue? It seems like it might be outdated?

**Comment by jessegreenberg** [comment]
I don't think this is issue is relevant anymore.

> (like moving the peer divs to match positions so zooming works well)

This feature in particular is going to be handled in #41, and whether or not it is off by default will be handled there.  But rendering the accessibility DOM and acesssibility features have not yet shown a negative impact on performance.

**Comment by samreid** [comment]
> But rendering the accessibility DOM and acesssibility features have not yet shown a negative impact on performance.

The main concern about performance was on iPad, has that been tested?

**Comment by jessegreenberg** [comment]
For the sims that have accessible content thus far (john-travoltage and balloons-and-static-electricity) we have tested performance on iPads, ```?profiler``` tool shows the same fps with and without current accessibility features.

Moving peer elements to match positions will likely be slower, I will revisit this issue after we investigate continue in phetsims/scenery#41.

**Comment by samreid** [comment]
Sounds good, thanks!
